Upload step for copy process completion added (#146)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/alerting-target.adoc

= Creating Alerting Email Templates

> Create a template for an email that is sent as an alert. Address the email to users within RPA Manager or to other persons that need to be aware of problems. Use keywords to extract the relevant information from the incident.

Reviewer: ```suggestion
Create a template for an email that is sent as an alert. Address the email to users within RPA Manager or to other people that must be aware of problems. Use keywords to extract the relevant information from the incident.
```

|Open the *Alerting* module
|Alerting Open

|Create, edit, and delete alerting targets and rules.
|Alerting Administration

|===

== Create Alert Targets

> Create a template for an email that is sent as an alert. Address the email to users within RPA Manager or to other persons that need to be aware of problems. Use keywords to extract the relevant information from the incident.

Reviewer: ```suggestion
Create a template for an email that is sent as an alert. Address the email to users within RPA Manager or to other persons that must be aware of problems. Use keywords to extract the relevant information from the incident.
```

Reviewer: global: persons > people

. In the *Alert Targets* view of the *Alerting* module, click *Create*.
. Complete the form *Create a new email target*:
+
* *Keywords*:
+

> Click on the *Keywords* icon to add placeholders to the subject or the body of the email. They are filled with the actual values at sending time.

Reviewer: ```suggestion
Click on the *Keywords* icon to add placeholders to the subject or the body of the email. The placeholders are filled with the actual values at sending time.
```
